### PR TITLE
[Windows, Doc] Backend mode missing doc

### PR DESCRIPTION
**Pull request by arifBurakDemiray**

### CODE REVIEW

**Review by turtledreams — CHANGES_REQUESTED** [review]

File: windows/next.md [hunk 1]
```diff
@@ -761,12 +819,296 @@ Segmentation segmentation = new Segmentation();
...
+segmentation.Add("campaign_user", "true");
+
+// with segmentation, firstView and timestamp
+Countly.Instance.BackendMode().StartView(DEVICE_ID, APP_KEY, "Login", "Desktop", true, segmentation, 1703752478530);</code></pre>
+<p>
+  <strong>Stop View</strong>
+</p>
+<pre><code class="csharp">Countly.Instance.BackendMode().StopView(string deviceId, string appKey, string name, string segment, long duration, Segmentation segmentations = null, long timestamp = 0);</code></pre>
+<p>
+  deviceId, appKey, name, segment and duration parameters are required. They should
```
[the remark this answers is not in the document]
Comment (arifBurakDemiray) [reply windows/next.md:932]: it also mandatory for events

**Review by turtledreams — APPROVED** [review]

**Comment by turtledreams** [comment]
tldr

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
&nbsp; needs to be removed

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
expand on the user profile custom modiffier descriptions
